Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md

```diff
@@ -68,6 +69,10 @@ All Cost Centers and Workspaces belong to an Organization.
 
 The administrators of an Organization can change many settings that affect the whole organizations. See more details [here](../Administering Your Organization/admin-overview.md).
 
+### Favorites
+
+The Favorites section contains a list of any workspace, cost center or organization that the user added for easier or more convenient access to. To add to this section, simply click the Star icon in the upper left hand corner of any of the aforementioned spaces. To remove, click the Star icon again.
```

Comment:
Hey @dsimon1992, what about mentioning that the star will appear highlighted in the items you have added to Favorites?

Reply:
Updated the description. Let me know if its suitable.